webpack/base.babel.js

```diff
@@ -172,6 +179,7 @@ export function webpackDefaults(env, config, bundles, isPlugin = false) {
           sourceMap: true,
           url: !config.isPlugin,
           ...cssLoaderOptions,
+          importLoaders: 2,
```

Comment:
I can tell you what it does, but I don't know why it's needed. The `importLoaders` option specifies the number of previous loaders that should be used (so it basically lets you disable the previous ones). At least that's my understanding of the option.

Example from the docs:

```javascript
module.exports = {
  module: {
    rules: [
      {
        test: /\.css$/i,
        use: [
          "style-loader",
          {
            loader: "css-loader",
            options: {
              importLoaders: 2,
              // 0 => no loaders (default);
              // 1 => postcss-loader;
              // 2 => postcss-loader, sass-loader
            },
          },
          "postcss-loader",
          "sass-loader",
        ],
      },
    ],
  },
};
```

Reply:
To be frank, this option has always confused me because by default it is 0, and that would imply that no other loaders would be used. That's a rather strange default. Therefore, it's quite possible that I am misunderstanding something.

Reply:
Seems like the confusion is universal :smile: https://github.com/webpack-contrib/css-loader/issues/228
So based on the discussion it seems like the we should probably keep this at 2.

Reply:
Well, at least we know what it's for. :D